docs/dqx/docs/guide.mdx

```diff
@@ -2,144 +2,126 @@
 sidebar_position: 3
 ---
 
+import Admonition from '@theme/Admonition';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 #  User guide
 
 ## Data Profiling and Quality Rules Generation
 
 Data profiling can be run to profile the input data and generate quality rule candidates with summary statistics.
-The generated rules/checks are input for the quality checking (see [Adding quality checks to the application](#adding-quality-checks-to-the-application)).
+The generated quality rules (checks) are input for the quality checking (see [Adding quality checks to the application](#adding-quality-checks-to-the-application)).
```

Comment:
Should we write the explanation that it's recommended to review generated checks?

Reply:
added